rel/files/mongooseimctl

```diff
@@ -264,9 +264,13 @@ ctl ()
 
 ctlexec ()
 {
+    export EMU=beam
```

Comment:
emulator beam? why not jit? Or maybe I'm understanding this flag wrong 🤔

Reply:
I copied these from the removed script. Actually both `PROGNAME` and `EMU` are set to the same values by default, so it's a bit redundant. See the source code:

https://github.com/erlang/otp/blob/master/erts/etc/common/erlexec.c